### PR TITLE
code review suggestions

### PR DESCRIPTION
**Pull request by hornunjb**
### Analysis

Was the program available in UC Github on time? 

* Yes, there was code for me to review on time

Is the program documented/commented well enough for you to understand?

* Small comments are present here and there, overall it was easy to understand and it followed a structure similar to that laid out in class which made it easier to understand.

Does the program compile? 

* Yes, program compiles and builds.

### Learned Technical Concepts

1. Proper class layout order via documentation
2. How I can better use live data and retrofit in my own project, following guidelines laid out in class and documentation
3. Better understanding of visibility modifiers and when/where to use them in my own projects and distinguishing between them

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -36,7 +36,7 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
     testImplementation 'androidx.arch.core:core-testing:2.0.0'
-    implementation 'com.squareup.retrofit2:retrofit:2.7.1'
-    implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
```
Comment (hornunjb) [inline app/build.gradle:39]: Made sure dependencies like retrofit are up to date
Source: https://nullbeans.com/9-reasons-for-keeping-software-dependencies-up-to-date/

File: app/src/main/java/com/brewfind/Service/BreweryService.kt [hunk 2]
```diff
@@ -9,7 +9,7 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class BreweryService {
-    fun fetchBreweries() : MutableLiveData<ArrayList<Brewery>> {
+    internal fun fetchBreweries() : MutableLiveData<ArrayList<Brewery>> {
```
Comment (hornunjb) [inline app/src/main/java/com/brewfind/Service/BreweryService.kt:12]: Updated visibility modifiers to specify internal type
Source: https://kotlinlang.org/docs/visibility-modifiers.html

File: app/src/main/java/com/brewfind/ui/main/MainFragment.kt [hunk 3]
```diff
@@ -29,4 +25,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
```
Comment (hornunjb) [inline app/src/main/java/com/brewfind/ui/main/MainFragment.kt:28]: Moved companion object to proper location (bottom)
Source: https://kotlinlang.org/docs/coding-conventions.html#source-file-names 
found under 'class layout' section

**Comment by discospiff** [comment]
Some good suggestions.  The group should review and consider a merge.

**Pull request merged**